packages/date/src/DateField.js

```diff
@@ -6,15 +6,19 @@ import Date from './Date';
 
 const DateField = ({ name, label, ...props }) => (
   <FormGroup for={name}>
-    {label && <Label for={name}>{label}</Label>}
+    {label && typeof label === 'string' ? (
```

Comment:
Looks like we missed some extra label attr spreading when adding in the new form library components.

The label type should look like the one linked below like reactstrap validation date:

https://github.com/Availity/availity-react/blob/master/packages/reactstrap-validation-date/src/AvDateRangeField.js#L129-L138